constant.json

```
@@ -0,0 +1,55 @@
+{
+    "id": "constant",
+    "summary": "Define a constant value",
+    "description": "Define a constant value that can be reused in multiple places of the process graph",
```

```suggestion
    "description": "Defines a constant value that can be reused in multiple places of a process.",
```

We don't use the process graph phrase in user communication any longer.

---

A dot is still missing at the end of the line ;-)